src/pip/_internal/wheel_builder.py

```diff
@@ -84,6 +91,21 @@ def should_build(
     return True
 
 
+def should_build_for_wheel_command(
+    req,  # type: InstallRequirement
+):
+    # type: (...) -> bool
+    return should_build(req, True, _always_true)
```

Comment:
I'd like to see the use of kwargs here, mostly to make it easier to understand what this call means, without needing to look up the definition of `should_build`.